css/parser/src/token.rs

```diff
@@ -1,6 +1,7 @@
 use logos::Logos;
 
-#[derive(Logos, Debug, PartialEq, Eq)]
+/// This does not implements [PartialEq] nor [Eq] as it's not optimized out.
```

Comment:
Not optimized out?

Reply:
Oh. It's wrong I guess.
PartialEq for an enum with fields is not optimized out.

Reply:
Oh, so you use `perf report` to optimize parser?

Reply:
I invoked `sudo perf record -a -g ../../target/release/deps/parser-xxxxxx` and looked at `perf report` to get name of the function which should be optimized.